### DISCUSSION TITLE
Re: Vercel hosting a domain, but pointing to Netlify

### DISCUSSION THREAD

**Discussion by bleue-bee** [vercel/community#3740] [Help] [answered]
### Summary

Hello, I transferred a domain to **Vercel** successfully, and built a NextJs app for an artist site... , but I was forced to use **Netlify** hosting due the cost _restrictions on images_ ...

I updated the name servers on Vercel to point to the name servers specified by Netlify.  The site is running as expected.  Then I received an email that there is an error and I need to point to Vercel  name space servers. Also my domain has not been added to  search engines after a month.

Questions:

- Do I understand that you can only use Vercel  hosting to use Vercel domains. To transfer in a top-level domain (apex domain) and use the domain,  you must use Vercel? If you could pls let me know if this is possible...
- If I can use **Vercel Domain hosting**, how can I update the name servers to point to Netlify w/out errors?
- I also noticed that no indexing has happened yet on the Domain transferred to Vercel. Is this because I am pointing to Netlify name servers? Is Vercel blocking my top-level domain from getting indexed?

Thanks! I appreciate the read. 

- I followed these examples from docs:
https://vercel.com/docs/projects/domains/add-a-domain
https://vercel.com/docs/projects/domains/add-a-domain#apex-domains

### Steps to Reproduce

- I followed these examples from docs:
https://vercel.com/docs/projects/domains/add-a-domain
https://vercel.com/docs/projects/domains/add-a-domain#apex-domains

**Comment by amyegan** [comment, marked as answer]
Hey @bleue-bee. You can definitely change nameservers or set A and CNAME records to other hosts if needed. We have a doc about [managing nameservers on Vercel](https://vercel.com/docs/projects/domains/managing-nameservers#) that shows how to do it.

> For domains registered with Vercel, you can [add custom nameservers](https://vercel.com/docs/projects/domains/managing-nameservers#add-custom-nameservers) to your Vercel-hosted domain, directly from the dashboard, allowing for delegation to other DNS providers. You can add up to four nameservers at once, and [revert to your previous settings](https://vercel.com/docs/projects/domains/managing-nameservers#restore-original-nameservers) if necessary.

If a project was mentioned in the email notification you received, then you should go into that project's settings and remove the domain from that project to avoid getting messages about it being improperly configured for the project.
 
Transferring the domain is another option. If the domain has been registered with Vercel for more than 60 days, then it is eligible under [ICANN rules](https://www.icann.org/resources/pages/text-2012-02-25-en#:~:text=Please%20note%20that%20you%20may,60%20days%20after%20a%20transfer) to transfer to a different registrar. The guide on [how do I transfer a domain out of Vercel](https://vercel.com/guides/how-do-i-transfer-my-domain-out-of-vercel) can walk you through the steps.

If you don't mind sharing more about the image cost restrictions you faced, I am happy to pass that along to the team. 🙂
